builder/openstack/run_config.go

```diff
@@ -51,8 +54,8 @@ func (c *RunConfig) Prepare(ctx *interpolate.Context) []error {
 		c.TemporaryKeyPairName = fmt.Sprintf("packer_%s", uuid.TimeOrderedUUID())
 	}
 
-	if c.UseFloatingIp && c.FloatingIpPool == "" {
-		c.FloatingIpPool = "public"
+	if c.FloatingIPPool != "" {
+		c.FloatingIPNetwork = c.FloatingIPPool
 	}
```

Comment:
There should be a check so not both are set.

Reply:
Yes, that makes sense, I will fix that.